spec/database_connection_spec.rb

```diff
@@ -0,0 +1,9 @@
+require './lib/database_connection.rb'
+
+describe Database_connection do
+  it "checks that data from airbnb_test database
+  deletes after an sql query runs" do
+    connect
+
+  end
+end
```

Comment:
I assume you have set this up for later, or have forgotten about it. At present it doesn't seem to be doing anything.